.github/workflows/fossa.yml

```diff
@@ -8,20 +8,25 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-go@v2
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Use Go ^1.14.x
+        uses: actions/setup-go@v2
+        with:
+          go-version: ^1.14.x
+
+      - name: Add GOPATH to GITHUB_ENV
+        run: echo "GOPATH=$(go env GOPATH)" >>"$GITHUB_ENV"
+
+      - name: Add GOPATH to GITHUB_PATH
+        run: echo "$GOPATH/bin" >>"$GITHUB_PATH"
+
+      - name: Run FOSSA scan and upload build data
+        uses: fossa-contrib/fossa-action@v1
```

Comment:
Why isn't this an official action under github.com/fossas ?

Reply:
Because this isn't an official action yet.